Support single runtime version

### Description

CI failed caused by `rtx` regexp error.
so support single runtime version for now

### Expected behavior

Install all runtime with `rtx` & CI passed

### Actual behavior

```
The application panicked (crashed).
Message:  error parsing regex: Syntax(
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
regex parse error:
    ^[3.11([-.].+)?$
     ^
error: unclosed character class
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
)
Location: src/tool.rs:368

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
chezmoi: exit status 101
Error: Process completed with exit code 1.
```

### Steps to reproduce

install multiple version with `rtx`

### Screenshots

_No response_

### Environment

macOS 13.6

### Additional information

_No response_

--- a/home/.chezmoidata.toml
+++ b/home/.chezmoidata.toml
@@ -13,15 +13,6 @@
     theme = "dracula"
     opacity = 0.75
 
-[rtx.versions]
-    # version should be in a array!!
-    node   = ["20"]
-    deno   = ["1"]
-    bun    = ["1"]
-    python = ["3.11", "3.12"]
-    # poetry = ["1.6"]
-    ruby   = ["3.2"]
-
 [plugins]
     vscode = [
         # Language

--- a/home/.chezmoiscripts/run_onchange_after_21-python.sh.tmpl
+++ b/home/.chezmoiscripts/run_onchange_after_21-python.sh.tmpl
@@ -3,9 +3,6 @@
 
 set -eo pipefail
 
-# Configure global version
-rtx use -g ruby@{{ .rtx.versions.python }}
-
 # Upgrade pip
 python3 -m pip install --upgrade pip
 python3 -m pip install pynvim

--- a/home/.chezmoiscripts/run_onchange_after_22-ruby.sh.tmpl
+++ b/home/.chezmoiscripts/run_onchange_after_22-ruby.sh.tmpl
@@ -3,9 +3,6 @@
 
 set -eo pipefail
 
-# Configure global version
-rtx use -g ruby@{{ .rtx.versions.ruby }}
-
 # Update gem
 gem update --system
 gem install bundler -N

--- a/home/.chezmoiscripts/run_onchange_after_23-nodejs.sh.tmpl
+++ b/home/.chezmoiscripts/run_onchange_after_23-nodejs.sh.tmpl
@@ -3,8 +3,4 @@
 
 set -eo pipefail
 
-# Configure global version
-rtx use -g node@{{ .rtx.versions.node }}
-rtx use -g deno@{{ .rtx.versions.deno }}
-
 pnpm add -g neovim

--- a/home/dot_config/rtx/config.toml.tmpl
+++ b/home/dot_config/rtx/config.toml.tmpl
@@ -5,9 +5,11 @@
 # NODE_ENV = 'production'
 
 [tools]
-{{- range $lang, $versions := .rtx.versions }}
-{{ $lang }} = [{{ range $i, $version := $versions | sortAlpha -}}{{- if $i -}}, {{ end -}}"{{ $version -}}"{{- end -}}]
-{{- end }}
+node   = "20"
+deno   = "1"
+bun    = "1"
+python = "3.11"
+ruby   = "3.2"
 
 # supports everything you can do with .tool-versions currently
 # nodejs = ['16', 'prefix:20', 'ref:master', 'path:~/.nodes/14']

